### ISSUE TITLE
Galaxy Pi Curriculum  - University of Zurich's "Swiss FinTech Innovation Lab" 

### ISSUE THREAD

**Issue by aseoconnor** [galaxypi/galaxy-growth#8]
We're going to be setting up a quick and dirty Galaxy node curriculum for the University of Zurich, where students will be given Galaxy nodes to access and learn blockchain related tasks using Galaxy.

It will be based out of their Fin Tech hub -  http://www.fintech.uzh.ch/en.html 

So the course should cover general blockchain/Galaxy side of things as well as on the Fintech side of blockchain/Galaxy that could be helpful and most likely will be used (assume i.e. focus on the nodes and simple currency we have atm and stuff down the road etc).

> Could be something as simple as get the node up and running, Get Galaxy up and running, sync, send / receive coin, students then work on token economics and then implement some basic rules based upon their proposed token economics...

Open question: Is this for a 1 hr class? A multi week endeavor? 

Thoughts on a scope and sequence: 
1. What is blockchain?
2. How does blockchain impact fintech
3. What problem is Galaxy solving? 
4. Set up Galaxy
5. What is a p2p payment system//how could galaxy facilitate

**Comment by aseoconnor** [comment]
@guylepage3 @clintnelsen @lukasetter I created an issue for the Zurich Curriculum

**Comment by guylepage3** [comment]
Great to see this getting started so quickly @aseoconnor 👍 

>Open question: Is this for a 1 hr class? A multi week endeavor?

I feel we should propose a multi-week course.

## Course Outline

#### Part I – Learn how to use blockchains
Using a Galaxy Pi Node
- Install and sync Ethereum
- Install and sync Galaxy
- Create address
- Send / receive coin

#### Part II – Token economics
- Students learn about token economics
- Develop their own token economics

#### Part III – Implement basic token
- Students then take their proposed token and implement it into their Galaxy Node.

**Comment by guylepage3** [comment]
Here's the only course available at the moment. (Use Google Chrome to translate page to English)
- https://studentservices.uzh.ch/uzh/anonym/vvz/index.html#/details/2018/003/SM/50876309

Here's an initial table of contents

## Table of contents
- General description
- Time frame
- Professor
- Course outline
- Learning outcome
- Course materials

**Comment by aseoconnor** [comment]
@guylepage3 who will be teaching this? 

I like the idea of 2-3 1 hour sessions. Also it seems like the uni is fairly technical, but that should be confirmed. 

The outline you've proposed makes sense =)

**Comment by guylepage3** [comment]
>who will be teaching this?

I believe Professor [Thomas Puschmann](https://studentservices.uzh.ch/uzh/anonym/vvz/index.html#/details/2018/003/P/01106625)

>it seems like the uni is fairly technical, but that should be confirmed.

Yes... Something we do not know. We should FLAG 🚩this one and make sure we find out more about the level of technical ability of the students.

**Comment by lukasetter** [comment]
@aseoconnor @guylepage3 

Few words to the above:

- While the course will be for graduate students at University of Zurich (which is by far overall less technical than ETH Zurich (ranked wolrd #7) who is almost purely a technical institution (think of MIT or Caltech) the class is out of said Fintech Innovation Lab and this is under the umbrella of the Institute for Informatics (IFI) as far as I know, hence on the more technical side of things.

- Also, am happy to reach out to Thomas again to see what scope exactly he had in mind (either filling a 2hrs lecture or a several lectures/weeks thing. At the same time I can find out what technical level those graduate students have - will shoot him an email now and get back as soon as he responds.

Let me know if further questions come up.

**Comment by lukasetter** [comment]
http://www.fintech.uzh.ch/en.html

Here on the left side the sub-chapters provide some background info to anticipate some of the direction of the course (and technical level).

**Comment by lukasetter** [comment]
Either way we definitely also need to "acquire" a technical class – maybe we can capture that at ETH ZH through their computer science students and our existing collaboration

**Comment by lukasetter** [comment]
Speaking of which:

**The rise of crypto in higher education**

_Key Findings_
42 percent of the world’s top 50 universities now offer at least one course on crypto or blockchain
Students from a range of majors are interested in crypto and blockchain courses — and universities are adding courses across a variety of departments
Original Coinbase research includes a Qriously survey of 675 U.S. students, a comprehensive review of courses at 50 international universities, and interviews with professors and students

https://blog.coinbase.com/the-rise-of-crypto-in-higher-education-81b648c2466f

**Comment by guylepage3** [comment]
>Here on the left side the sub-chapters provide some background info to anticipate some of the direction of the course (and technical level).

Yeah I was using their current curriculum as a guide.

**Comment by guylepage3** [comment]
Yeah. I feel that we should now put what we have so far into a quick v1 proposal document. I'm happy to push this doc out the door initially.

@aseoconnor are you familiar with Adobe InDesign for documentation?

**Comment by guylepage3** [comment]
I'll be creating the Galaxy curriculum proposal v1 today... https://github.com/galaxypi/galaxy-design/issues/25

**Comment by lukasetter** [comment]
**Gents, got response from Thomas re open questions:**

"Hi Lukas,

Thanks for your note.
I included my comments in the email below (in **bold**).

Kind regards,
Thomas

On 29 Aug 2018, Lukas Etter wrote:


_Hi Thomas

We already started outlining a potential curriculum as discussed yesterday.

For this reason a couple of questions arose:_

_- what should be the scope of our materials/input? How frequent/long is the class taught? Do you have a 1h class on Galaxy / setting up a node/P2P money transfer in mind OR content that could spread over several classes? Some rough outline/guideline would be very helpful for us._

**> The scope should be about your solution/Galaxy; what is it all about and what could the students contribute. We have planned an intensive 2 day lab session on Dec. 13th and 14th where the students develop their own concepts and solutions. Maybe we could include a 1h session about Galaxy on one of these days so that the students could use Galaxy to setup a node?** 

_- the class is more economics/business focused than IT, right? So rather cryptoeconomics content than the techstack behind blockchains/Galaxy? what is overall technical level of the students?_

**> The class is mixed from IT, business, banking and finance, economics and law students. Most of them should have an IT background as well.**


Hope this helps @guylepage3 @aseoconnor @faddat

**Comment by guylepage3** [comment]
Not sure who's who in this message @lukasetter Can you please provide a bit of clarity?

**Comment by lukasetter** [comment]
Bold and top sentence is Thomas - rest (italic and normal) is from me

**Comment by aseoconnor** [comment]
@guylepage3 I'm very familiar with adobe illustrator... But maybe best to do Google slides for team collaboration and then we can send as a PDF 

And it seems like they are giving us an hour - so we should just focus on Galaxy Pi. If they want a blockchain 101 primer I could throw that together as well

**Comment by guylepage3** [comment]
@aseoconnor Yeah.. I designed a Pages doc for the community so that everyone can use it. https://github.com/galaxypi/galaxy-design/issues/26

**Comment by guylepage3** [comment]
>And it seems like they are giving us an hour - so we should just focus on Galaxy Pi. If they want a blockchain 101 primer I could throw that together as well

Yeah.. Ok. so now that we know a bit more.. Here's what I am proposing..

## Course Outline

In this one hour course you'll learn how to install and run a Galaxy full node and what problems the Galaxy network solves and why you should be running a Galaxy node. You'll also get a quick introduction to blockchains; what they're good at and what they're bad at. 

#### Part I – Quick introduction to blockchains 101
- What is a blockchain
- What they're good at
- What they're bad at 

#### Part II – Why Galaxy matters
- What is Galaxy
- Why you should run and maintain a Galaxy node

#### Part III
- Installing, running and maintaining a Galaxy Pi Node
    - Install and sync Ethereum
    - Install and sync Galaxy
    - Create address
    - Send / receive coin

**Comment by guylepage3** [comment]
<img width="763" alt="screenshot 2018-08-31 18 02 00" src="https://user-images.githubusercontent.com/1711854/44937600-00f35e00-ad48-11e8-80ed-31c16a861443.png">
<img width="763" alt="screenshot 2018-08-31 18 02 11" src="https://user-images.githubusercontent.com/1711854/44937601-00f35e00-ad48-11e8-9366-e8a1c0d1031a.png">

**Comment by lukasetter** [comment]
Looks super nice @guylepage3! Think makes sense to send this brief outline to Thomas once during this week to see if it roughly covers what he had in mind and where we should cut/elaborate.


Here few comments on the doc:

**Part I:**
- looks good to me
- maybe touch on few common use cases (e.g. getting rid of interemediary, logistics supply chain, etc)

**Part II:**
- What is galaxy, how does it work, what is is good for
- what is will (potentially) be able to do in future (IOT, AI, Edge/Fog computing)

**Part III:**
- do we even want to touch on installing and syncing Ethereum? (also they would need the hard drive for that and not just the pi)
- installing and syncing Galaxy would probably come before Ethereum client?!
- probably, if included at all, would out the Ethereum part at the very end of that chapter, even after the create wallet and and send coin



Also, "Call for ideas/improvements" could be interesting?

**Comment by guylepage3** [comment]
Great feedback @lukasetter, I'll add this to my list of to do's for today.. I agree that we should send this out to Thomas after I make the edits for his feedback. I'll ping you to send it over...

I'm working on this now.

**Comment by guylepage3** [comment]
![university-of-zurich-v1 1 0_page_1](https://user-images.githubusercontent.com/1711854/44997391-e86b8980-af7b-11e8-8f89-1c0caaf430bb.png)
![university-of-zurich-v1 1 0_page_2](https://user-images.githubusercontent.com/1711854/44997392-e86b8980-af7b-11e8-9030-9c78e457d6f9.png)

**Comment by guylepage3** [comment]
Hey @lukasetter here's the course curriculum for Thomas' feedback.. Please send this over to him when you get a moment. Thank you.
[university-of-zurich-v1.1.0.pdf](https://github.com/galaxypi/galaxy-growth/files/2346261/university-of-zurich-v1.1.0.pdf)

**Comment by aseoconnor** [comment]
Looks really good Guy!

After we get feedback from U Zurich we can build out the curriculum for the course

**Comment by guylepage3** [comment]
Thanks @aseoconnor...

@lukasetter, here's the revised version for Thomas without the languages section.

[university-of-zurich-v1.1.1.pdf](https://github.com/galaxypi/galaxy-growth/files/2347230/university-of-zurich-v1.1.1.pdf)

**Comment by lukasetter** [comment]
Great, thanks a lot, sent and waiting for feedback now.
